virttest/utils_net.py

```diff
@@ -4467,3 +4468,19 @@ def delete_linux_bridge_tmux(linux_bridge_name, iface_name=None, ignore_status=F
     else:
         cmd = 'ip link delete %s' % linux_bridge_name
     return utils_misc.cmd_status_output(cmd, shell=True, verbose=True, ignore_status=ignore_status)
+
+
+def ensure_default_network():
```

Comment:
I think utils_net module should be focus on system network, so how about adding it to utils_libvirt.libvirt_network?

Reply:
Updated, Thank you!